tests/CMakeLists.txt

```diff
@@ -248,6 +248,18 @@ set_tests_properties(test-help-with-other-options
 set_tests_properties(test-help-with-notexisting-options
                      PROPERTIES PASS_REGULAR_EXPRESSION  "aktualizr command line options")
 
+add_test(NAME test-legacy-interface-with-notexisting-file
+         COMMAND aktualizr -c  ${PROJECT_SOURCE_DIR}/config/config.toml.example --legacy-interface nonexistentfile)
+
+set_tests_properties(test-legacy-interface-with-notexisting-file
+                     PROPERTIES PASS_REGULAR_EXPRESSION  "Legacy external flasher not found: nonexistentfile")
+
+add_test(NAME test-secondary-config-with-notexisting-file
+         COMMAND aktualizr -c  ${PROJECT_SOURCE_DIR}/config/config.toml.example --secondary-config nonexistentfile)
+
+set_tests_properties(test-secondary-config-with-notexisting-file
+                     PROPERTIES PASS_REGULAR_EXPRESSION  "nonexistentfile is not exists!!!")
```

Comment:
This looks pretty good, just a couple minor grammar issues I'd like to fix:
1) The error message should probably be "does not exist!".
2) The test names should be "-with-nonexistent-file".

Reply:
Ok, thank You

Reply:
Done